src/app-bar.jsx

```diff
@@ -260,7 +268,7 @@ const AppBar = React.createClass({
         }
 
         menuElementLeft = (
-          <div style={prepareStyles(Object.assign({}, styles.iconButtonStyle))}>
+          <div style={prepareStyles(iconLeftStyle)}>
```

Comment:
Yeah, but this is a breaking change. you shouldn't remove the `styles.iconButtonStyle` just override it: `Object.assign({}, styles.iconButtonStyle, iconLeftStyle)`